### ISSUE TITLE
Project GARDEN Subsite Update-- Compensation Text Changes

### ISSUE THREAD

**Issue by ianchandlercampbell** [lookit/lookit-api#1263]
# TL;DR
On https://childrenhelpingscience.com/garden/, we need to update all mentions of participant compensation to clarify that they will receive a $5 gift card each time they complete one of studies #1-7, and a $15 gift card for completing study #8. The current text discusses this as a bonus for completion, which is now inaccurate. This will result in **5 changes to the subsite "copy"**

# Narrative

1. Main Page: Change "Each study will provide a $5 gift card, and families that do all 8 of our first studies will receive a bonus $10 gift card." to "You will receive a $5 gift card for each study for studies #1-7 and a $15 gift card for study #8."
2. About Page: Under "How does participating in Project GARDEN work?", change "If you complete all of the first set of 8 studies, you will receive an extra gift card as a thank you." to "You will receive a $5 gift card for each study for studies #1-7, and a $15 gift card for study #8."
3. How to Join Page: Under "Is Project GARDEN for me?", change "Would like to receive a $5 Amazon.com (U.S.-based) gift card each time they participate in one of our 8 studies." to "Would like to receive a $5 Amazon.com (U.S.-based) gift card for each study for studies #1-7, and a $15 gift card for study #8." 
4. How to Join Page: Under "5. Click on the first Project GARDEN study...", change "All eligible families will receive a $5 Amazon.com gift card (U.S.-only) within 7 business days after participating in each study." to "All eligible families will receive an Amazon.com gift card (U.S.-only) within 7 business days after participating in each study." 
5. How to Join Page: Under "5. Click on the first Project GARDEN study...", change "Families who complete all 8 studies will receive an extra $10 Amazon.com gift card! In total, families who complete all 8 studies will receive $50 in Amazon.com gift cards (eight $5 gift cards and one extra $10 gift card)." to "Families who complete all eight studies will receive a total of $50 in Amazon.com gift cards (seven $5 gift cards and one $15 gift card)."

# Acceptance Criteria
A precise description of how/when this issue will be considered closed, e.g.:

- [ ] All 5 changes have been implemented on the live subsite
